app/models/post.rb

```diff
@@ -0,0 +1,18 @@
+class Post < ApplicationRecord
+  has_many :comments
+  has_many :likes
+  belongs_to :users, foreign_key: 'author_id', class_name: 'User'
```

Comment:
- [ ] This should be `belongs_to :user` as well you can have a look at this [link](https://guides.rubyonrails.org/association_basics.html)